pipelines/azure-pipelines.yml

```diff
@@ -115,6 +115,7 @@ jobs:
 
       - task: notice@0
         displayName: "NOTICE File Generator"
+        continueOnError: "true"
```

Comment:
>continueOnError: "true" [](http://example.com/codeflow?start=8&length=23)

condition: not(eq(variables['Build.Reason'], 'PullRequest'))

Reply:
changed to suggested